docker-entrypoint.sh

```diff
@@ -57,6 +60,9 @@ then
   echo "Alert Database password secret set; using value from secret."
   alertDatabasePassword=$(cat $dockerSecretDir/ALERT_DB_PASSWORD)
   export ALERT_DB_PASSWORD=$alertDatabasePassword;
+
+  alertDatabaseAdminPassword=$alertDatabasePassword;
```

Comment:
Is this set to the appropriate value? Shouldn't it use the Admin user?

Reply:
see the comment above